src/pybind/mgr/volumes/pathspec.py

```diff
@@ -0,0 +1,87 @@
+import os
+
+class SubvolumeSpec(object):
+    """
+    Specification of a subvolume, identified by (subvolume-id, group-id) tuple. Add fields as
```

Comment:
"Add fields as required...". What does that mean?

Reply:
in the sense, add fields that identify a (unique) subvolume.